src/test/java/bio/terra/common/DaoUtilsTest.java

```diff
@@ -27,12 +26,12 @@ public void testOrderByClause() {
     assertThat(
         "order by clause looks correct",
         DaoUtils.orderByClause(EnumerateSortByParam.NAME, SqlSortDirection.ASC, "foo"),
-        equalTo(" ORDER BY foo.name asc "));
+        equalTo(" ORDER BY foo.name ASC "));
```

Comment:
Maybe this should instead ignore case?
```suggestion
        equalToIgnoringCase(" ORDER BY foo.name ASC "));
```